lib/ClaimVerifier.ts

```diff
@@ -32,6 +34,7 @@ export class ClaimVerifier {
     const filteredRoles = roles.filter(Boolean) as {
```

Comment:
Why to cast to `{name; namespace; status}`?

Reply:
@JGiter I tried removing this code and it results in some abnormalities in returned `verifiedRoles`. Maybe I give it some more time and remove this in a separate PR? So that the UI in `iam-client-examples` could pick this informative rejection changes ASAP. @jrhender what are your thoughts?

Reply:
> Maybe I give it some more time and remove this in a separate PR

I agree, better to do it in a separate PR